bindings/python/examples/gyroid_module.py

```diff
@@ -15,7 +15,11 @@
  """
 
 import math
+import sys
 import numpy as np
+
+sys.path.append("/Users/k/projects/python/badcad/wip/manifold/build/bindings/python")
```

Comment:
When testing locally, I set my `CMAKE_INSTALL_PREFIX` to be in a folder that Python checks when searching for packages; it might help avoid needing to modify the PATH?

Reply:
Whoops missed that. Yeah I realized after I could also set `export PYTHONPATH=build/bindings/python` to get the same result. 

I'd like to figure out why the `pip install` is like 4x slower than direct cmake build, it looked like it runs `git` every time when I watch `top`